services/device-registry-base/src/main/java/org/eclipse/hono/deviceregistry/service/device/AutoProvisioner.java

```diff
@@ -221,8 +215,12 @@ public void setConfig(final AutoProvisionerConfigProperties config) {
                     false,
                     false);
 
-            final DownstreamSender sender = senderTracker.result();
-            return sender.sendAndWaitForOutcome(msg, span.context())
+            return protonBasedDownstreamSender.sendEvent(tenantTracker.result().getPayload(),
+                        new RegistrationAssertion(deviceId),
+                        EventConstants.CONTENT_TYPE_EMPTY_NOTIFICATION,
```

Comment:
this should now be the dedicated content-type for the well-known *device registration* event ...

Reply:
sorry, wrong PR ;-) This will be tackled later ...

Reply:
Yes, but since we created a separate issue for that I also implemented this via a separate PR: https://github.com/eclipse/hono/pull/2423